handler/experience.go

```
@@ -0,0 +1,34 @@
+
+func MakeExperienceHandler() ExperienceHandler {
+	return ExperienceHandler{
+		content: "./storage/fixture/experience.json",
+	}
+}
+
+func (h ExperienceHandler) Handle(w baseHttp.ResponseWriter, r *baseHttp.Request) *http.ApiError {
+	fixture, err := os.ReadFile(h.content)
+
+	if err != nil {
+		slog.Error("Error reading experience file: %v", err)
```

golangci-lint (1.64.8)

slog: slog.Error arg "err" should be a string or a slog.Attr (possible missing key or value)

(govet)